src/components/Forms.js

```diff
@@ -33,28 +33,30 @@ export const TextArea = props => {
 };
 
 export const Input = props => {
-  
   const {
     className,
     onChange,
     placeholder,
     readOnly,
     value,
     icon,
+    type,
   } = props;
 
   const styles = classNames({
-    [className]: className
+    [className]: className,
   });
 
   const allowedProps = Object.keys(props).reduce((acc, prop) => {
-    const filteredProps = {...acc};
-    if (!!['onKeyPress', 'onKeyDown', 'onKeyUp', 'onFocus'].find(p => p === prop)) {
+    const filteredProps = { ...acc };
+    if (
+      !!["onKeyPress", "onKeyDown", "onKeyUp", "onFocus"].find(p => p === prop)
```

Comment:
instead of using find here you could write
`["onKeyPress", "onKeyDown", "onKeyUp", "onFocus"].includes(prop)`